pymc3/step_methods/smc.py

```diff
@@ -28,7 +28,7 @@
 from .arraystep import metrop_select
 from ..backends import smc_text as atext
 
-__all__ = ['SMC', 'ATMIP_sample']
+__all__ = ['SMC', 'smc_sample']
```

Comment:
@twiecki Sure? Shouldn't it be `sample_smc`? I think this goes clearly against the common convention.